QuanLiSoTietKiem/Views/Shared/_Layout.cshtml

```diff
@@ -1,42 +1,364 @@
+                                    <a href="index3.html">Dashboard 3</a>
+                                </li>
+                                <li>
+                                    <a href="index4.html">Dashboard 4</a>
+                                </li>
+                            </ul>
+                        </li>
+                        <li class="d-flex align-content-center ">
+                            <i class="fas fa-chart-bar" style="padding: 20px 0"></i>
+                            <a>
+                                @Html.ActionLink("Quản lí nhân viên", "Index", "Staff", new { @class = "w-100 ml-3" })
+                            </a>
+
+
+                        </li>
+                        <li class="d-flex align-content-center ">
+                            <i class="fas fa-table" style="padding: 20px 0"></i>
+
+                            @Html.ActionLink("Quản lí chức vụ", "index", "Sach", new { @class = "w-100 ml-3" })
+
+                        </li>
+                        <li class="d-flex align-content-center ">
+                            <i class="far fa-check-square" style="padding: 20px 0"></i>
+
+                            @Html.ActionLink("Quản lí lãi suất", "index", "Sach", new { @class = "w-100 ml-3" })
+
+                        </li>
+                        <li class="d-flex align-content-center ">
+                            <i class="fas fa-calendar-alt" style="padding: 20px 0"></i>
+
+                            @Html.ActionLink("Quản lí kì hạn", "index", "Sach", new { @class = "w-100 ml-3" })
+
+                        </li>
+                        <li class="d-flex align-content-center ">
+                            <i class="fas fa-map-marker-alt" style="padding: 20px 0"></i>
+
+                            @Html.ActionLink("Quản lí báo cáo", "index", "Sach", new { @class = "w-100 ml-3" })
+
+                        </li>
+                        <li class="d-flex align-content-center ">
+                            <i class="fas fa-copy" style="padding: 20px 0"></i>
+
+                            @Html.ActionLink("Quản lí Khách hàng", "index", "Sach", new { @class = "w-100 ml-3" })
+
+                        </li>
+
+                        <li class="d-flex align-content-center ">
+                            <i class="fas fa-magic" style="padding: 20px 0"></i>
+
+                            @Html.ActionLink("Quản lí tài khoản nhân viên đã quá hạn", "index", "Sach", new { @class = "w-100 ml-3" })
+
+                        </li>
+
+                    </ul>
+                </nav>
+            </div>
+        </aside>
+        <!-- END MENU SIDEBAR-->
+        <!-- PAGE CONTAINER-->
+        <div class="page-container">
+
+
+            <!-- HEADER DESKTOP-->
+            <header class="header-desktop">
+                <div class="section__content section__content--p30">
+                    <div class="container-fluid">
+                        <div class="header-wrap">
```

Comment:
Xoá cái dòng này phần Search for data, để lại phần tên + logout và account setting thôi